experimental/SetPartitions/src/Util.jl

```diff
@@ -0,0 +1,95 @@
+"""
+    helper_new_point_values_vector(p::Vector{Vector{Int}}, q::Vector{Vector{Int}})
+
+Return a semantically identical partition in form of an vector, 
+which has new number values.
+"""
+function helper_new_point_values_vector(p::Vector, q::Vector)
+
+    p_points = vcat(copy(p[1]), copy(p[2]))
+
+    if !(isempty(p_points))
+        new_id, index = findmax(p_points)
+        new_id += 1
+    else
+        new_id = 1 
+    end
+    
+    q_points = vcat(copy(q[1]), copy(q[2]))
+    new_ids = Dict()
```

Comment:
Not type stable. Perhaps like this (I am guessing the types
```suggestion
    new_ids = Dict{Int,Int}()
```